config/secrets.yaml.template

```diff
@@ -1,15 +1,22 @@
 # For instructions on creating a Batch and Storage account, see
 # Getting Started (https://github.com/Azure/aztk/blob/master/docs/00-getting-started.md)
 # NOTE - YAML requires a space after the colon. Ex: "batchaccountname: mybatchaccount"
-batch:
-    batchaccountname: 
-    batchaccountkey: 
-    batchserviceurl: 
 
-storage:
-    storageaccountname: 
-    storageaccountkey: 
-    storageaccountsuffix: core.windows.net
+servicePrincipal:
```

Comment:
Should be changed to serviceprincipal for consistency. (in general, this file should probably have underscores to separate words like cluster.yaml does, but that is a breaking change so I'm not sure we want to go through with that).

Reply:
well this is already a breaking change this new format so we could go for it

Reply:
I don't think this is a breaking change. We will still support the existing format, and the new service principal format.

If we only add underscores for the service principal format, we wouldn't break anything but then we have a inconsistent syntax within the file.